### PR TITLE
Add amoled/black theme

### PR DESCRIPTION
**Pull request by mueller-ma**
Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

File: mobile/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -34,6 +34,29 @@
         <item name="colorPrimaryDark">@color/blue_grey_900</item>
     </style>
 
+    <style name="openHAB.Amoled" parent="openHAB.DayNight">
+        <item name="chartTheme">black</item>
+        <item name="android:windowBackground">@color/black</item>
```
Comment (mueller-ma) [inline mobile/src/main/res/values/themes.xml:39]: I'm not sure if changing the charts and background from dark grey to black is enough. I'll ask in the forum for feedback.

**Comment by maniac103** [comment]
What's the reasoning for wanting to add this?
I still see zero point whatsoever in an 'Amoled' theme. People think it provides battery savings, but those savings aren't even worth thinking about, at least if you aren't staring at the OH app all day, which seems highly unlikely. IMHO, the light/dark theme toggle is totally sufficient.

**Comment by mueller-ma** [comment]
A user requested the theme [here](https://community.openhab.org/t/android-app-beta/41382/155) and two users via Play Store review. For wall mounted tablets there is some battery saving.

**Comment by maniac103** [comment]
Is assuming wall mounted tablets operate on battery instead of AC a reasonable assumption?

**Comment by mueller-ma** [comment]
Yes, it is. But saving energy isn't that bad, right? And even if it doesn't save any energy, it looks better if the display is of bad quality.
I won't use it, but IMO an additional entry in the list doesn't hurt. Let's see what feedback we get from the user who requested the theme.

File: mobile/src/main/res/values/themes.xml (hunk 1 above)
Comment (maniac103) [reply mobile/src/main/res/values/themes.xml:39]: I think you'll want to at least change the card background for frames. I think it's `#111` by default.

Comment (mueller-ma) [reply mobile/src/main/res/values/themes.xml:39]: I'm not sure what you mean. The app doesn't use cards for frames, only basic ui does this.

Comment (maniac103) [reply mobile/src/main/res/values/themes.xml:39]: Never mind, I thought we are using cards for the frames.

**Pull request merged**